lib/ddr/models/solr_document.rb

```diff
@@ -243,6 +248,15 @@ def get_pid(field)
         ActiveFedora::Base.pid_from_uri(get(field)) rescue nil
       end
 
+      def inherited_research_help_contact
+        if admin_policy_pid
+          query = ActiveFedora::SolrService.construct_query_for_pids([admin_policy_pid])
+          results = ActiveFedora::SolrService.query(query)
+          doc = results.map { |result| ::SolrDocument.new(result) }.first
+          doc[Ddr::IndexFields::RESEARCH_HELP_CONTACT].first
+        end
```

Comment:
This is a note that I'd like to refactor and use `admin_policy` here, but we can merge as is.